src/TouristInfoCards.jsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+
+const TouristInfoCard = () => {
+  return (
+    <div className="cards">
+      <div className="card">
+        <img
+          src="https://a.cdn-hotels.com/gdcs/production95/d16/cc514640-8f0f-11e8-9bad-0242ac110002.jpg"
+          alt="Glasgow city center"
+          className="card-img"
+        />
+        <h1>Glasgow</h1>
+        <p className="card-paragraph">
+          Glasgow is a port city on the River Clyde in Scotland's western
+          Lowlands. It's famed for its Victorian and art nouveau architecture, a
+          rich legacy of the city's 18th–20th-century prosperity due to trade
+          and shipbuilding. Today it's a national cultural hub, home to
+          institutions including the Scottish Opera, Scottish Ballet and
+          National Theatre of Scotland, as well as acclaimed museums and a
+          thriving music scene.
+        </p>
+        <div className="card-btn-container">
+          <a href="http://peoplemakeglasgow.com/" target="_blank">
+            <button className="card-btn">More information</button>
+          </a>
+        </div>
+      </div>
```

Comment:
There is a lot of duplication in this file, what can we do when we see duplicated html with different values?